Mathlib/MeasureTheory/Function/LpSeminorm.lean

```diff
@@ -895,93 +915,95 @@ theorem snorm_map_measure (hg : AEStronglyMeasurable g (Measure.map f μ)) (hf :
 
-theorem MeasurableEmbedding.snorm_map_measure {g : β → F} (hf : MeasurableEmbedding f) :
+theorem _root_.MeasurableEmbedding.snorm_map_measure {g : β → F} (hf : MeasurableEmbedding f) :
     snorm g p (Measure.map f μ) = snorm (g ∘ f) p μ := by
   by_cases hp_zero : p = 0
   · simp only [hp_zero, snorm_exponent_zero]
   by_cases hp : p = ∞
   · simp_rw [hp, snorm_exponent_top]
-    exact hf.ess_sup_map_measure
+    exact hf.essSup_map_measure
   · simp_rw [snorm_eq_lintegral_rpow_nnnorm hp_zero hp]
     rw [hf.lintegral_map]
+    rfl
 #align measurable_embedding.snorm_map_measure MeasurableEmbedding.snorm_map_measure
 
-theorem MeasurableEmbedding.memℒp_map_measure_iff {g : β → F} (hf : MeasurableEmbedding f) :
+theorem _root_.MeasurableEmbedding.memℒp_map_measure_iff {g : β → F} (hf : MeasurableEmbedding f) :
     Memℒp g p (Measure.map f μ) ↔ Memℒp (g ∘ f) p μ := by
-  simp_rw [mem_ℒp, hf.ae_strongly_measurable_map_iff, hf.snorm_map_measure]
+  simp_rw [Memℒp, hf.aestronglyMeasurable_map_iff, hf.snorm_map_measure]
 #align measurable_embedding.mem_ℒp_map_measure_iff MeasurableEmbedding.memℒp_map_measure_iff
 
-theorem MeasurableEquiv.memℒp_map_measure_iff (f : α ≃ᵐ β) {g : β → F} :
+theorem _root_.MeasurableEquiv.memℒp_map_measure_iff (f : α ≃ᵐ β) {g : β → F} :
     Memℒp g p (Measure.map f μ) ↔ Memℒp (g ∘ f) p μ :=
-  f.MeasurableEmbedding.memℒp_map_measure_iff
+  f.measurableEmbedding.memℒp_map_measure_iff
 #align measurable_equiv.mem_ℒp_map_measure_iff MeasurableEquiv.memℒp_map_measure_iff
 
-omit mβ
-
 end MapMeasure
 
 section Trim
 
-theorem snorm'_trim (hm : m ≤ m0) {f : α → E} (hf : strongly_measurable[m] f) :
+theorem snorm'_trim (hm : m ≤ m0) {f : α → E} (hf : StronglyMeasurable[m] f) :
     snorm' f q (ν.trim hm) = snorm' f q ν := by
   simp_rw [snorm']
   congr 1
   refine' lintegral_trim hm _
-  refine' @Measurable.pow_const _ _ _ _ _ _ _ m _ (@Measurable.coe_nnreal_ennreal _ m _ _) _
-  apply @strongly_measurable.measurable
-  exact @strongly_measurable.nnnorm α m _ _ _ hf
+  refine' @Measurable.pow_const _ _ _ _ _ _ _ m _ (@Measurable.coe_nnreal_ennreal _ m _ _) q
+  apply @StronglyMeasurable.measurable
+  exact @StronglyMeasurable.nnnorm α m _ _ _ hf
 #align measure_theory.snorm'_trim MeasureTheory.snorm'_trim
 
-theorem limsup_trim (hm : m ≤ m0) {f : α → ℝ≥0∞} (hf : measurable[m] f) :
+theorem limsup_trim (hm : m ≤ m0) {f : α → ℝ≥0∞} (hf : Measurable[m] f) :
     (ν.trim hm).ae.limsup f = ν.ae.limsup f := by
   simp_rw [limsup_eq]
   suffices h_set_eq : { a : ℝ≥0∞ | ∀ᵐ n ∂ν.trim hm, f n ≤ a } = { a : ℝ≥0∞ | ∀ᵐ n ∂ν, f n ≤ a }
   · rw [h_set_eq]
   ext1 a
-  suffices h_meas_eq : ν { x | ¬f x ≤ a } = ν.trim hm { x | ¬f x ≤ a }
-  · simp_rw [Set.mem_setOf_eq, ae_iff, h_meas_eq]
-  refine' (trim_measurable_set_eq hm _).symm
+  suffices h_meas_eq : ν { x | ¬f x ≤ a } = ν.trim hm { x | ¬f x ≤ a } by
+    simp_rw [Set.mem_setOf_eq, ae_iff, h_meas_eq]; rfl
```

Comment:
Any reason you changed this?

Reply:
`; rfl` is because Lean can't close the goal without this.
`by` is changed in the process of my trial and error; I've reverted.